.github/workflows/docker.yml

```diff
@@ -0,0 +1,40 @@
+name: Docker Image CI
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
```

Comment:
Remove this before this is merged, as you only want to push to AWS on merges to the main branch